simde/x86/avx512/abs.h

```diff
@@ -524,7 +524,7 @@ simde_mm512_mask_abs_ps(simde__m512 src, simde__mmask16 k, simde__m512 v2) {
 SIMDE_FUNCTION_ATTRIBUTES
 simde__m512d
 simde_mm512_abs_pd(simde__m512d v2) {
-  #if defined(SIMDE_X86_AVX512F_NATIVE) && (!defined(HEDLEY_GCC_VERSION) || HEDLEY_GCC_VERSION_CHECK(8,3,0))
+  #if defined(SIMDE_X86_AVX512F_NATIVE) && (!defined(HEDLEY_GCC_VERSION) || (HEDLEY_GCC_VERSION_CHECK(7,4,0) && !HEDLEY_GCC_VERSION_CHECK(8,0,0)) || HEDLEY_GCC_VERSION_CHECK(8,3,0))
```

Comment:
Can you move this logic to `simde/simde-common.h` by defining a `SIMDE_BUG_GCC_87467` there?

See https://github.com/simd-everywhere/simde/blob/fdbd6b2616e8ec2509c3ad7087c2e4d36c0a5f5d/simde/simde-common.h#L973-L976 and onwards

Reply:
Ah, I see. I will do so. Thanks!